app/main.ipynb

```diff
@@ -449,41 +427,58 @@
     "matrix2 = [[5, 6], [7, 8]]"
    ],
    "metadata": {
-    "collapsed": false,
-    "ExecuteTime": {
-     "end_time": "2023-07-03T13:36:06.776877Z",
-     "start_time": "2023-07-03T13:36:06.757338Z"
-    }
-   }
+    "collapsed": false
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-11-05T01:04:03.487621Z",
+     "start_time": "2024-11-05T01:04:03.465367Z"
+    }
+   },
    "cell_type": "code",
-   "execution_count": 27,
-   "outputs": [],
    "source": [
-    "# write your numpy code here and estimate the time of execution"
+    "matrix1 = np.array([[1, 2], [3, 4]])\n",
+    "matrix2 = np.array([[5, 6], [7, 8]])\n",
+    "\n",
+    "def unknown_signature(matrix1, matrix2):\n",
+    "    return matrix1 @ matrix2\n",
+    "\n",
+    "%time unknown_signature(matrix1, matrix2)"
```

Comment:
The task requires element-wise multiplication of matrices, but the current implementation uses matrix multiplication (`@`). Use `np.multiply(matrix1, matrix2)` for element-wise multiplication.